mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -224,6 +224,20 @@ private[clustering] trait LDAParams extends Params with HasFeaturesCol with HasM
   /**
    * For Online optimizer only: [[optimizer]] = "online".
    *
+   * @group expertParam
+   */
+  @Since("2.3.0")
+  final val epsilon = new DoubleParam(this, "epsilon", "(For online optimizer)" +
+    " A (positive) learning parameter that controls the convergence of variational inference.",
+    ParamValidators.gt(0))
```

Comment:
what about using `0.0` to remark that this is a Double?

Reply:
Yes, use 0.0 is good. Because other Double in this class is using 0, to keep the same code style, I also use 0 here.
It is ok to change all 0 to 0.0.